<ui version="4.0" >
 <class>Ui::EqualizerDialog</class>
 <widget class="QDialog" name="Ui::EqualizerDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>화면 설정하기</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>밝기</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="horizontalSlider_5" >
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>65535</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QSpinBox" name="brightness_spin" >
     <property name="accelerated" >
      <bool>true</bool>
     </property>
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>65535</number>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>대비</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSlider" name="horizontalSlider_6" >
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>65535</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QSpinBox" name="contrast_spin" >
     <property name="accelerated" >
      <bool>true</bool>
     </property>
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>65535</number>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>색조</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QSlider" name="horizontalSlider_8" >
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>65535</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QSpinBox" name="hue_spin" >
     <property name="accelerated" >
      <bool>true</bool>
     </property>
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>65535</number>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>채도</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QSlider" name="horizontalSlider_9" >
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>65535</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QSpinBox" name="saturation_spin" >
     <property name="accelerated" >
      <bool>true</bool>
     </property>
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>65535</number>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="default_button" >
       <property name="text" >
        <string>기본값</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset_button" >
       <property name="text" >
        <string>초기화</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_button" >
       <property name="text" >
        <string>닫기</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>Ui::EqualizerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>264</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel" >
     <x>270</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brightness_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>135</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel" >
     <x>246</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>contrast_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>168</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>228</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hue_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>139</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturation_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>113</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel" >
     <x>223</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturation_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>235</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel" >
     <x>116</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hue_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>229</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel" >
     <x>148</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrast_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>222</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel" >
     <x>149</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brightness_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>225</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>173</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
